### PR TITLE
mockup of the Services page

### PR DESCRIPTION
**Pull request by june-andronick**
New services page.

### CODE REVIEW

**Comment by june-andronick** [comment]
I think I'm done with this final draft. To note:
- Cog and DW have a separate page on the Foundation website
- Breakaway doesn't (but we haven't ask Benno yet; just took blurb from announcement)

Next step would be to write to the 3 of them with screenshots and ask for feedback. Given I'm away next week, sooner is better ;)

**Comment by june-andronick** [comment]
Also: I haven't added AOS yet

**Comment by gernotheiser** [comment]
I get a 404 on the preview link

**Comment by lsf37** [comment]
Preview link [here](https://htmlpreview.github.io/?https://github.com/seL4/website_pr_hosting/blob/PR_30/localhost/index.html)

**Comment by gernotheiser** [comment]
Services page looks good (but I'd like AOS added before we go live ;-)
Also, the bullet list should link to the relevant sections.

The individual companies' pages they will probably want to change, now that the formatting is defined, they don't look so great. I suggest sending them not just a screen shot but the actual HTML source so they can play with it themselves.

**Review by lsf37 — APPROVED** [review]
Happy with the current state, pending approval by the members.

File: content/Foundation/Services/dornerworks.pml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <tr>
+      <td style="padding:15px">
+	<a href="https://dornerworks.com">
+	  <img src="/Foundation/Membership/LOGOS/DornerWorks.svg" style="width: 200px" alt="DornerWorks Ltd">
+	</a>
+      </td>
+      <td>
+
+         DornerWorks helps product makers turn their ideas into
+         realitywith FPGA, hardware, and embedded software engineering
```
Comment (Furao) [inline content/Foundation/Services/dornerworks.pml:24]: add a space between reality and with

File: content/Foundation/Services/dornerworks.pml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+<li>
+  seL4 Microkernel
+  <ul>
+  <li> Configuration, integration, customization, and system optimization
+  </li>
+  </ul>
+</li>
+<li>
+  Guest OS and hardware portingVirtualization and hypervisor distributions
+</li>
```
Comment (Furao) [inline content/Foundation/Services/dornerworks.pml:56]: Should be:

```
<li>
  Guest OS and hardware porting
</li>
<li>
  Virtualization and hypervisor distributions
</li>

**Review by Furao — CHANGES_REQUESTED** [review]
Just a couple of text issues. I appreciate the endorsement process page. That will be helpful.

File: content/Foundation/Services/dornerworks.pml (hunk 1 above)
Comment (lsf37) [reply content/Foundation/Services/dornerworks.pml]: Thanks for spotting these, now fixed!

File: content/Foundation/Services/dornerworks.pml (hunk 2 above)
Comment (lsf37) [reply content/Foundation/Services/dornerworks.pml]: Done!

**Comment by lsf37** [comment]
(reverted accidental push)

**Comment by june-andronick** [comment]
I've added AOS as an endorsed training.
I've deleted commented out sections.
I changed the order and names in the drop-down menus. 
Cog and Breakaway have accepted the text for their parts.
This is ready for review and merge.

**Review by gernotheiser — APPROVED** [review]

**Pull request merged**